Content.Server/Chat/Systems/ChatSystem.cs

```diff
@@ -957,19 +962,21 @@ public sealed class EntitySpokeEvent : EntityEventArgs
     public readonly EntityUid Source;
     public readonly string Message;
     public readonly string? ObfuscatedMessage; // not null if this was a whisper
+    public readonly EntityUid? TelepathyTarget; // not null if this was a telepathic message to a specific mob
 
```

Comment:
Hardcoding in a specific entity for this is a bad idea and is just going to make chat more of a mess.

Given sadpanda is working on fixing chat code to not be a dog's breakfast this PR can't be merged until then as it will conflict too much.

Reply:
Ah, awesome! Looking forward to the chat refactor going in. I'll revisit this when that happens.